TLM/TLM/UI/SubTools/SpeedLimits/Overlay/SpeedLimitsOverlay.cs

```diff
@@ -492,6 +492,19 @@ private struct CachedSegment {
             return ret;
         }
 
+        /// <summary>
+        /// Determines whether to show green default speed icon or themed icon for normal speed override view.
+        /// </summary>
+        /// <param name="drawSpeed">The speed to show.</param>
+        /// <param name="defaultSpeed">The default speed limit.</param>
+        /// <returns>Returns <c>true</c> = use default signs, or <c>false</c> = use themed signs.</returns>
+        private bool UseUnthemedIcons(SpeedValue? drawSpeed, SpeedValue? defaultSpeed) {
+            return
+                !drawSpeed.HasValue
+                || (Options.differentiateDefaultSpeedsInNormalView
+                    && drawSpeed.Value.Equals(defaultSpeed));
```

Comment:
Should I check for `.HasValue` here?

Reply:
In this case, no because condition is "does not have value" OR (must have) and optionCheck is true and is Equal...

Reply:
I meant for `defaultSpeed` but I assume that always has value?

Also, why isn't it `drawSpeed.Value.Equals(defaultSpeed.Value)` ?

Would there be merit to adding an operator override on `SpeedValue` so we could do `drawSpeed == defaultSpeed`?

Reply:
By default it uses reflection to compare `structs` so == should work already if I'm not mistaken  (just a tiny bit slower)

`drawSpeed.Value.Equals(defaultSpeed.Value)` will throw exception if there is no value :)